app-backend/api/src/main/scala/stac/Routes.scala

```diff
@@ -72,122 +72,137 @@ trait StacRoutes
-                    p.hasPrevious,
-                    p.hasNext,
-                    p.page,
-                    p.pageSize,
-                    p.results.map(signExportUrl(_))
-                )
-              )
-              .transact(xa)
-              .unsafeToFuture
-          }
-      }
-    }
-  }
-
-  def createStacExport: Route = authenticate { user =>
-    authorizeScope(ScopedAction(Domain.StacExports, Action.Create, None), user) {
-      entity(as[StacExport.Create]) { newStacExport =>
-        newStacExport match {
-          case StacExport
-                .AnnotationProjectExport(_, _, _, annotationProjectId) =>
-            authorizeAsync {
-              AnnotationProjectDao
-                .authorized(
-                  user,
-                  ObjectType.AnnotationProject,
-                  annotationProjectId,
-                  ActionType.View
-                )
-                .map(_.toBoolean)
+  def listStacExports: Route =
+    authenticate { user =>
+      authorizeScope(
+        ScopedAction(Domain.StacExports, Action.Read, None),
+        user
+      ) {
+        (withPagination & stacExportQueryParameters) {
+          (page: PageRequest, params: StacExportQueryParameters) =>
+            complete {
+              StacExportDao
+                .list(page, params, user)
+                .map(
+                  p =>
+                    PaginatedResponse[StacExport.WithSignedDownload](
+                      p.count,
+                      p.hasPrevious,
+                      p.hasNext,
+                      p.page,
+                      p.pageSize,
+                      p.results.map(signExportUrl(_))
+                  ))
                 .transact(xa)
                 .unsafeToFuture
             }
-          case StacExport.CampaignExport(_, _, _, campaignId) =>
-            authorizeAsync {
-              CampaignDao
-                .authorized(
-                  user,
-                  ObjectType.Campaign,
-                  campaignId,
-                  ActionType.View
-                )
-                .map(_.toBoolean)
+        }
+      }
+    }
+
+  def createStacExport: Route =
+    authenticate { user =>
+      authorizeScope(
+        ScopedAction(Domain.StacExports, Action.Create, None),
+        user
+      ) {
+        entity(as[StacExport.Create]) { newStacExport =>
+          newStacExport match {
+            case StacExport
+                  .AnnotationProjectExport(_, _, _, annotationProjectId) =>
+              authorizeAsync {
+                AnnotationProjectDao
+                  .authorized(
+                    user,
+                    ObjectType.AnnotationProject,
+                    annotationProjectId,
+                    ActionType.View
+                  )
+                  .map(_.toBoolean)
+                  .transact(xa)
+                  .unsafeToFuture
+              }
+            case StacExport.CampaignExport(_, _, _, _, campaignId) =>
```

Comment:
This was the only line I changed in this file, FYI. The rest is wacky formatting changes from `scalafmt`.